linuxgsm.sh

```diff
@@ -15,12 +15,13 @@
 
 # Debugging
 if [ -f ".dev-debug" ]; then
-	exec 5>dev-debug.log
+	exec 5>>dev-debug.log
 	BASH_XTRACEFD="5"
 	set -x
+	echo "invocation linuxgsm.sh $@ $(date "+%Y-%m-%d-%H:%M:%S:%N")"
```

Comment:
Required or not, IF we want to keep it this might be better (at least according to codacy):
```suggestion
	echo "invocation linuxgsm.sh $* $(date "+%Y-%m-%d-%H:%M:%S:%N")"
```